### ISSUE TITLE
Is the bot ready?

### ISSUE THREAD

**Issue by niknaylos** [nwarsalee/discord_faceithub_bot#1]
Hey, I'm currently running Faceit Hub. Looking for good discord related tools for it here and there. 
Have couple of questions:

1. Is this bot ready yet? 
2. Do you have any setup/function documentation for it? 
Would love to use the bot 

lmk cause I'd love to use this.

**Comment by nwarsalee** [comment]
Hey, appreciate your interest!

1. It is ready. We've tested it on one of our servers with a group of people and it has worked pretty well so far (there will definitely be some bugs so please let us know if you encounter any) There are still some features that we would like to add but as I said it currently can work. We actually have an instance of the bot running now. I can get you the invite link if you would like to try it out.

2. As of right now we don't have any documentation on setup but I can write up a quick guide soon. It is not very complicated to setup but there are some initial setups within the discord setup that is needed.

If you do happen to try it out and have recommendations and/or bugs that you would like to report please don't hesitate to make a ticket, we'll try and get it fixed when we can.

Thanks!

**Comment by niknaylos** [comment]
Yeah, alright, I will let you know if we are going to use it, is there any way I could contact you outside github? (discord perhaps)

**Comment by nwarsalee** [comment]
My discord - Warsna#4581

**Comment by bboychris168** [comment]
@nwarsalee I added you on discord for some questions about the bot. my name on discord is MayNgu#8666
